include/pcmki/pcmki_verify.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2023 the Pacemaker project contributors
+ *
+ * The version control history for this file may have further details.
+ *
+ * This source code is licensed under the GNU Lesser General Public License
+ * version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+ */
+
+#ifndef PCMK__PCMKI_PCMKI_VERIFY__H
+#  define PCMK__PCMKI_PCMKI_VERIFY__H
+
+#include <crm/common/output_internal.h>
+#include <crm/common/scheduler.h>
+#include <pcmki/pcmki_transition.h>
+#include <crm/cib.h>                    // cib_t
+#include <pacemaker.h>
+#include <stdbool.h>
+#include <stdint.h>
```

Comment:
I believe the last 5 headers aren't needed for this one, but libxml/tree.h is (for xmlNodePtr)

Basically, a header file should include any other header files needed to define all types etc. used in the including header, Here, output_internal.h is needed for `pcmk__output_t` and scheduler.h is needed for `pcmk_scheduler_t`.